Microsoft.HealthVault.RestApi/Generated/ActionPlanObjectives.cs

```diff
@@ -60,7 +60,7 @@ public ActionPlanObjectives(MicrosoftHealthVaultRestApi client)
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        /// <exception cref="MicrosoftKhronosCloudCommonErrorsErrorResponseException">
```

Comment:
We corrected the long type names so "ErrorResponseException" is what it should be

---
In reply to: [121519936](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/54#discussion_r121519936) [](ancestors = 121519936)